### PR TITLE
Capture Model And Recon Data and send Through Client

### PR DESCRIPTION
**Pull request by aniehuser**
in this commit:
- Added exit path for `YwSaveException`
- Added Dto's and update `RunDto`
- surface model data from `Run.model` object
- surface recon data from `ReconFacts` class
- fixed a bug where log files path from `@log` annotation were not properly referenced 

This commit is waiting for the server to be able to handle the final request body shape
This commit still needs some testing

### CODE REVIEW

File: src/main/java/org/yesworkflow/save/data/ChannelDto.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import org.yesworkflow.model.Channel;
+
+public class ChannelDto
+{
+    public Integer id;
+    public Integer outPort;
+    public Integer inPort;
+    public Long data;
+    public Boolean isInflow;
+    public Boolean isOutlfow;
```
Comment (clundin55) [inline src/main/java/org/yesworkflow/save/data/ChannelDto.java]: typo isOutlfow -> isOutflow

File: src/main/java/org/yesworkflow/save/data/ChannelDto.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+    public ChannelDto(Integer id, Integer outPort, Integer inPort, Long data, Boolean isInflow, Boolean isOutlfow) {
```
Comment (clundin55) [inline src/main/java/org/yesworkflow/save/data/ChannelDto.java]: same typo

**Comment by clundin55** [comment]
https://cdn-images-1.medium.com/max/1600/1*QQnxLRyGkTqHamqONvrftA.png

**Comment by aniehuser** [comment]
> https://cdn-images-1.medium.com/max/1600/1*QQnxLRyGkTqHamqONvrftA.png

papa bless for intellij command + n

**Pull request merged**